docs/application/native/guides/applications/component-based-app.md

```diff
@@ -0,0 +1,642 @@
+        <icon>@DESKTOP_ICON@</icon>
+        <app-control id="dial">
+            <operation name="http://tizen.org/appcontrol/operation/dial"/>
+            <mime name="*"/>
+        </app-control>
+        <app-control id="dial-for-excel">
+            <operation name="http://tizen.org/appcontrol/operation/dial"/>
+            <mime name="application/vnd.ms-excel"/>
+        </app-control>
+    ```
+
+2.  Registering an action
+
+    To receive action events, each component instance should register the action.
+    You can registering an action as follows.
+    ```
+    static void __frame_component_action_cb(component_h context,
+            const char *action, app_control_h app_control,
+            void *user_data)
+    {
+    }
+
+    static Evas_Object *__frame_component_create_cb(component_h context, void *user_data)
+    {
+        component_register_action(context, "dial-for-excel")
+        return frame_get_window(ad->frame);
+    }
+
+    component_class_h app_control_component_add(component_class_h comp_class,
+            const char *component_id, void *user_data)
+    {
+        frame_component_lifecycle_callback_s callback = {
+            .create = __frame_component_create_cb,
+            .action = __frame_component_action_cb,
+        };
+
+        return component_based_app_add_frame_component(comp_class,
+                component_id, &callback, user_data);
+    }
+    ```
+
+3.  Sending an action event to another application
+
+    Actions are declared in the manifest files. To send a proper action, you should carefully set app-control values according to the manifest file app-control specification. In this example, the app-control will be set for "dial-for-excel" action.
+    ```
+    static int __app_control_send(const char *app_id, const char *component_id,
+            void *user_data)
+    {
+        app_control_h handle = NULL;
+        int ret;
+
+        ret = app_control_create(&handle);
+        if (ret != APP_CONTROL_ERROR_NONE)
+            return ret;
+
+        ret = app_control_set_operation(handle, "http://tizen.org/appcontrol/operation/dial");
+        if (ret != APP_CONTROL_ERROR_NONE) {
+            app_control_destroy(handle);
+            return ret;
+        }
+
+        ret = app_control_set_mime(handle, "application/vnd.ms-excel");
+        if (ret != APP_CONTROL_ERROR_NONE) {
+            app_control_destroy(handle);
+            return ret;
+        }
+
+        ret = app_control_send_launch_request_async(handle,
+                __app_control_result_cb, NULL, user_data);
+        if (ret != APP_CONTROL_ERROR_NONE) {
+            app_control_destroy(handle);
+            return ret;
+        }
+        return 0;
+    }
+
+    static void __button_clicked_cb(void *data, Evas_Object *obj,
+        void *event_info) {
+        __app_control_send("org.tizen.base-component", "base-frame", NULL);
+    }
+    ```
+
+
+<a name="launch_application"></a>
+## Launch a Component-Based Application
```

Comment:
Avoid usage of articles in the heading. Please remove "a".

Reply:
Done